java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_2/NumberInTheRangeValidation.java

```diff
@@ -0,0 +1,16 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class NumberInTheRangeValidation implements Validator {
+    private int birthMonth;
```

Comment:
The class name is `NumberInTheRangeValidation` and the instance field name - `birthMonth`. The idea was the instance field will be named like `fieldName` and the class will be called like this:

```
new NumberInTheRangeValidation("birthMonth"), new NumberInTheRangeValidation("birthDay")...
```

you could even add possible ranges as additional arguments of the constructor